### ISSUE TITLE
Oder-Eingabefelder sind noch nicht richtig positioniert.

### ISSUE THREAD

**Issue by DongEnergy** [EkoDondav/rehaklex#35]
Ja, die Anordnung der Eingabefelder bei Buy und Sell scheint auch vertauscht zu sein:

![selection_017](https://cloud.githubusercontent.com/assets/4712140/15907317/f5e5c640-2dbc-11e6-900d-6c51cc8ac4b9.png)

**Comment by EkoDondav** [comment]
Igitt! Liegt wohl an deinem Browser? Welchen hast Du? Etwa IE11?? Sollte so aussehen:
![image](https://cloud.githubusercontent.com/assets/10418916/15907697/79aea5f4-2dbe-11e6-854d-f37e63b8e7d5.png)

**Comment by McCompexpert** [comment]
Ich benutze moderne browser ;-))
![selection_020](https://cloud.githubusercontent.com/assets/4712140/15908227/df004118-2dc0-11e6-9eb9-25f48cfbf1f9.png)

**Comment by EkoDondav** [comment]
Sehr gut :-) Benutzt Chrome (ist schneller) -- muss alles natürlich auch in FF gehen. Welche FF Version hat die Börse? IE11 werde ich per alert ablehnen.

**Comment by McCompexpert** [comment]
Ja, man muss sich auf einen Browser begrenzen. Chrome ist gut. Es gibt unterschiedliche Versionen von FF: in Citrix - eine, wenn jemand ein Notebook hat, dann andere, neuere :-)). Ich installiere mir mal Chrome auf Ubuntu

**Comment by McCompexpert** [comment]
Yep. mit Chrome is alles gut

![selection_021](https://cloud.githubusercontent.com/assets/4712140/15908886/3096e470-2dc4-11e6-9671-e9c0f47322fe.png)

**Comment by DongEnergy** [comment]
Gut, dann benutze ich Chrome

On 08.06.2016, at 21:59, McCompexpert notifications@github.com wrote:

Yep. mit Chrome is alles gut

**Comment by EkoDondav** [comment]
Werde es für FF etwas verbessern, denn die Börse hatunter Citrix nur FF als IE11-Alternative. IE11 wird ab jetzt gesperrt sein. (es kommt 'ne error page).

**Comment by DongEnergy** [comment]
Sehr gut!

On 08.06.2016, at 22:45, EkoDondav notifications@github.com wrote:

Werde es für FF etwas verbessern, denn die Börse hatunter Citrix nur FF als IE11-Alternative. IE11 wird ab jetzt gesperrt sein. (es kommt 'ne error page).
